osmosis-1/osmosis-1.assetlist.json

```diff
@@ -176,6 +176,43 @@
         "OSMO": 15
       }
     },
+    {
+      "description": "The native Token of Medas Digital.",
+      "denom_units": [
+        {
+          "denom": "ibc/01e94a5ff29b8ddefc86f412cc3927f7330e9b523cc63a6194b1108f5276025c",
+          "exponent": 0,
+          "aliases": [
+            "umedas"
+          ]
+        },
+        {
+          "denom": "medas",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/01e94a5ff29b8ddefc86f412cc3927f7330e9b523cc63a6194b1108f5276025c",
+      "name": "MedasDigital",
+      "display": "medas",
+      "symbol": "MEDAS",
+      "ibc": {
+        "source_channel": "channel-519",
+        "dst_channel": "channel-0",
+        "source_denom": "umedas"
+      },
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/medasdigital/images/medas.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/medasdigital/images/medas.svg"
+      },
+      "keywords": [
+        "osmosis-main",
```

Comment:
please remove 'osmosis-main' (requires Osmosis governance to add to Main site) and remove 'osmosis-info' (which requires >=$10,000 USD liquidity on chain), and please move to Medas to the bottom of the list.